### ISSUE TITLE
[8.0] Kibana `GET /api/stats` & `GET /api/status` breaking changes

### ISSUE THREAD

**Issue by afharo** [elastic/beats#27569]
**Describe the enhancement:**
Kibana is planning to remove the `process` info structure from its metrics (https://github.com/elastic/kibana/issues/104124). This will likely affect Metricbeat's Kibana stats and status modules.

**Describe a specific use case for the enhancement or feature:**
The reasoning for removing that piece of info from the responses is detailed in the issue: https://github.com/elastic/kibana/issues/104124.

TL;DR: For 8.0, we are planning on adding support to Kibana running multiple forks in the same host. This means that `process` is unlikely to be meaningful anymore.
